frontend/src/forms/Registry/index.tsx

```diff
@@ -12,17 +12,22 @@ import { RequireNoSuffix, RequirePrefix, ValidatorName, ValidatorRequired } from
 import { Prompt } from "widgets/Prompt";
 import { REGISTRY_FORM_ID } from "../formIDs";
 import sc from "utils/stringConstants";
+import { CustomizedButton } from "widgets/Button";
+import { KPanel } from "widgets/KPanel";
 
 const styles = (theme: Theme) =>
   createStyles({
-    root: {},
+    root: {
+      padding: 20,
```

Comment:
should replace it with `theme.spacing(2)`, theme.spacing(2) would be 16px
padding/margin should use `theme.spacing` as possible as you can.